api/users/views.py

```diff
@@ -496,21 +496,21 @@ class UserDraftRegistrations(JSONAPIBaseView, generics.ListAPIView, UserMixin):
     required_read_scopes = [CoreScopes.USERS_READ, CoreScopes.NODE_DRAFT_REGISTRATIONS_READ]
     required_write_scopes = [CoreScopes.USERS_WRITE, CoreScopes.NODE_DRAFT_REGISTRATIONS_WRITE]
 
-    serializer_class = DraftRegistrationSerializer
+    serializer_class = DraftRegistrationSerializerLegacy
     view_category = 'users'
     view_name = 'user-draft-registrations'
 
     ordering = ('-modified',)
 
     def get_queryset(self):
-        user = self.get_user()
-        node_qs = Node.objects.get_nodes_for_user(user, permissions.ADMIN_NODE)
-        return DraftRegistration.objects.filter(
-            Q(registered_node__isnull=True) |
-            Q(registered_node__is_deleted=True),
-            branched_from__in=list(node_qs),
-            deleted__isnull=True,
-        )
+        requested_user = self.get_user()
```

Comment:
Remove `requested_user` - Does not offer meaningful functionality

Reply:
🆗